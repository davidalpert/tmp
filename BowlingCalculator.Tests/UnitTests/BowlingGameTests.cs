using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using NUnit.Framework;
using ObjectApproval;

namespace BowlingCalculator.Tests.UnitTests
{
    [TestFixture]
    public class BowlingGameTests
    {
        [Test]
        public void Can_add_players()
        {
            var game = new BowlingGame(1);

            game.SetPlayerName(1, "Mal");

            ObjectApprover.VerifyWithJson(game.ListPlayerNames());
        }

        [Test]
        public void Can_add_several_players()
        {
            var game = new BowlingGame(4);

            game.SetPlayerName(1, "Mal");
            game.SetPlayerName(2, "Inara");
            game.SetPlayerName(3, "Wash");
            game.SetPlayerName(4, "Zoe");

            ObjectApprover.VerifyWithJson(game.ListPlayerNames());
        }
    }
}
