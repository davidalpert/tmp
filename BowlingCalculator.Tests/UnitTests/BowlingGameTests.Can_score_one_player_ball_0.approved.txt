{
    "CurrentPlayer": "Mal",
    "CurrentFrame": 1,
    "CurrentBall": 1
}